### ISSUE TITLE
:not(:lang(...)) not supported?

### ISSUE THREAD

**Issue by papandreou** [dperini/nwsapi#18]
I ran into some weird `:not(...)` selectors on Slack's homepage (in https://a.slack-edge.com/acdbf/style/rollup-marketing.css) that nwsapi doesn't support.

Repro:

```js
new (require('jsdom').JSDOM)('').window.document.querySelector(':not(:lang(ja))')
```

```
SyntaxError: unknown pseudo-class selector ':not(:lang(ja))'
    at emit (/home/andreas/work/assetgraph/node_modules/nwsapi/src/nwsapi.js:529:17)
    at compileSelector (/home/andreas/work/assetgraph/node_modules/nwsapi/src/nwsapi.js:1171:17)
    at compile (/home/andreas/work/assetgraph/node_modules/nwsapi/src/nwsapi.js:718:19)
    at collect (/home/andreas/work/assetgraph/node_modules/nwsapi/src/nwsapi.js:1409:18)
    at _querySelectorAll (/home/andreas/work/assetgraph/node_modules/nwsapi/src/nwsapi.js:1349:18)
    at Object._querySelector [as first] (/home/andreas/work/assetgraph/node_modules/nwsapi/src/nwsapi.js:1283:14)
    at DocumentImpl.<anonymous> (/home/andreas/work/assetgraph/node_modules/jsdom/lib/jsdom/living/nodes/ParentNode-impl.js:65:42)
    at DocumentImpl.querySelector (/home/andreas/work/assetgraph/node_modules/jsdom/lib/jsdom/utils.js:123:45)
    at Document.querySelector (/home/andreas/work/assetgraph/node_modules/jsdom/lib/jsdom/living/generated/Document.js:689:45)
```

It works fine in Firefox and Chrome with `document.querySelector`.

**Comment by dperini** [comment]
@papandreou 
validation for the selector reported in the issue `:not(:lang(ja))` has been fixed by this commit f0024f66b23143ea73f50d753ee2e33a75f6128b

I am sure the resolver for the :lang(xx) pseudo-class was already passing the few tests I have so it should work, the problem is that many specific tests are still missing both in my repository and in jsdom.

We are working on improve this aspect to avoid future regression.
It would be nice if you can give feedback and test the fix in your environment,

Thank you for reporting the issue.

**Comment by papandreou** [comment]
Thanks a lot for fixing it, that was fast :tada:

**Comment by Zirro** [comment]
This got fixed in version 2.0.7.
